components/esp32/gdbstub.c

```diff
@@ -285,12 +388,84 @@ static int gdbHandleCommand(unsigned char *cmd, int len) {
 		gdbPacketEnd();
 	} else if (cmd[0]=='?') {	//Reply with stop reason
 		sendReason();
-	} else {
+#if configUSE_TRACE_FACILITY == 1
+	} else if (cmd[0]=='H' && reenteredHandler != -1) { //Continue with task
+		if (cmd[1]=='g' || cmd[1]=='c') {
+			const char * ret = "OK";
+			data++;
+			i=gdbGetHexVal(&data, -1);
+			reenteredHandler = 1; //Hg0 is the first packet received after connect
```

Comment:
Suggest to rename `reenteredHandler` since actually it is used to indicate that we already entered the handler.